packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -25,14 +25,29 @@ import { Fragment, useCallback, useEffect, useState } from 'react';
 import ReactMarkdown from 'react-markdown';
 import remarkBreaks from 'remark-breaks';
 import { utils } from 'ethers';
-import { useAppDispatch } from '../../hooks';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { nounVotesForProposalQuery } from '../../wrappers/subgraph';
+import { useQuery } from '@apollo/client';
+import VoteProgresBar, { ProgressBarVariant } from '../../components/VoteProgressBar';
+import proposalStatusClasses from '../../components/ProposalStatus/ProposalStatus.module.css';
+import NounImageVoteTable from '../../components/NounImageVoteTable';
 
 dayjs.extend(utc);
 dayjs.extend(timezone);
 dayjs.extend(advanced);
 
 const AVERAGE_BLOCK_TIME_IN_SECS = 13;
 
+// Helper function to transform response from graph into flat list of nounIds that voted
```

Comment:
TSDoc.org comes in handy here. Will help extract the content elsewhere when needed.

```
/**
 * Helper function to transform response from graph into flat list of nounIds that voted supportDetailed for the given prop
 *
 * @param data - lorem ipsum
 * @param supportDetailed - lorem ipsum
 * @returns - lorem ipsum
 */
```
or simply: 
```
/**
 * Helper function to transform response from graph into flat list of nounIds that voted supportDetailed for the given prop
 */
```